.github/workflows/build-public-ami.yml

```
@@ -43,6 +39,13 @@ jobs:
           fi
         shell: bash
 
+      - name: Set whether to skip ami creation in packer
+        run: |
+          if [ "${{ github.event_name }}" == "workflow_dispatch" ]; then
+            echo "Setting SKIP_CREATE_AMI to False
```

missing the end quote
```suggestion
            echo "Setting SKIP_CREATE_AMI to False"
```

---

good catch, it's been a long day.